docs/UserGuide.md

```diff
@@ -236,6 +236,45 @@ Perhaps for some reason, you decide not to go through with an application. Maybe
 
 :information_source: Note the successful command execution message in the system feedback box upon successful deletion of the company.
 
+## Use Case #3: Editing a company or a role <a id="usecase3"></a>
+Say you received an email for an interview and the point of contact's email is different.
+Here's how you can make changes to the application:
+1. [Edit the company](#c-edit-c) from the company list.  
```

Comment:
Like @kerbysoh mentioned in my previous PR, maybe it'd be good to include a link from the feature to this use case like such: 

<img width="546" alt="image" src="https://user-images.githubusercontent.com/77252831/161116201-f6fff1cb-bce6-4555-975b-952a1e719707.png">

Reply:
ok done. Thanks!